lib/rules/newline-per-chained-call.js

```diff
@@ -38,6 +38,19 @@ module.exports = {
 
         var sourceCode = context.getSourceCode();
 
+        /**
+         * Gets the property text of a given MemberExpression node.
+         * If the text is multiline, this returns only the first line.
+         *
+         * @param {ASTNode} node - A MemberExpression node to get.
+         * @returns {string} The property text of the node.
+         */
+        function getPropertyText(node) {
+            var prefix = node.computed ? "[" : ".";
```

Comment:
Do we possibly want a suffix (at least for when the property name is single-like) as well?